modules/nextflow/src/main/groovy/nextflow/cli/CmdLog.groovy

```diff
@@ -61,7 +61,7 @@ class CmdLog extends CmdBase implements CacheBase {
     static final public NAME = 'log'
 
     @Parameter(names = ['-s'], description='Character used to separate column values')
-    String sep = '\t'
+    String sep = 'TAB'
 
```

Comment:
Not understanding this change

Reply:
Have a look at the print screen that I attached to the pull request body text. Even though in the source code it's clearly a tab symbol (`\t`), in the CLI nothing appears as the default value. It could be a number of different things. My change tells the user it's a tab. I don't know if this is the best solution, but we should think on something to put there instead of the current solution.

Type `nextflow log -h` and you'll see it's not clear what's the default value for the `-s` option.

Reply:
Maybe escaping the `\`? `\\t`?

Reply:
Oh, only now I realized. This is probably used somewhere, not only for the `-h` of the command line, right? 😅 My bad. How can we solve this then?

Reply:
ahh, now understand yes `\\t`!